fluidBuild.config.cjs

```diff
@@ -77,14 +77,18 @@ module.exports = {
 		"build:test:cjs": ["typetests:gen", "tsc", "api-extractor:commonjs"],
 		"build:test:esm": ["typetests:gen", "build:esnext", "api-extractor:esnext"],
 		"api": {
-			dependsOn: ["api-extractor:commonjs", "api-extractor:esnext"],
+			dependsOn: ["api-extractor:commonjs", "api-extractor:esnext", "api-extractor:report"],
```

Comment:
If we aren't using API-Extractor to generate the rollups, can't we omit the first two deps here?